modules/sbo-rbac-requirements.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// *openshift-docs/applications/connecting_applications_to_services/exposing-binding-data-from-a-service
+[id="sbo-rbac-requirements_{context}"]
+= RBAC requirements
+
+To expose the backing service binding data using the {servicebinding-title}, you require certain Role-based access control (RBAC) permissions. You must specify certain verbs under the `rules` field of the `ClusterRole` resource to allow the RBAC persmissions for the backing service resources. Defining these `rules` enables the {servicebinding-title} with permissions to read the binding data of the the backing service resources throughout the cluster.
+
+The {servicebinding-title} performs requests against Kubernetes API using a dedicated service account. By default this account has permissions to bind services to workloads, both represented by the following standard Kubernetes objects:
+
+* `Deployments`
+* `DaemonSets`
+* `ReplicaSets`
+* `StatefulSets`
+* `DeploymentConfigs`
+
+The Operator service account is bound to an aggregated cluster role, allowing Operator providers or cluster administrators to enable binding custom service resources to workloads. You can enclose the required permissions within a `ClusterRole` by labeling it with the `servicebinding.io/controller` flag and setting the flag value to `true`.
+
+The following example shows how to allow the {servicebinding-title} to `read`, `watch`, and `list` the custom resources (CRs) of Crunchy PostgreSQL Operator by enclosing the required permissions within a `ClusterRole` resource:
```

Comment:
About `read`: The example includes `get`, can it be confusing to the users? It's clear that read=get, but maybe seeing 2 options can make users doubt which one to use?

Reply:
@missmesss It is `get`. My bad! Rectifying it. Thanks :)